src/WebJobs.Extensions.DurableTask/EntityScheduler/RequestMessage.cs

```diff
@@ -81,7 +88,7 @@ public void SetInput(object obj)
                 }
                 else
                 {
-                    this.Input = MessagePayloadDataConverter.Default.Serialize(obj);
+                    this.Input = this.dataConverter.MessageConverter.Serialize(obj);
                 }
```

Comment:
Shouldn't we be using this data converter in the two `GetInput` methods below? @sebastianburckhardt

Reply:
Yes, I would think so also. Not sure why we are not.